helix-term/src/commands.rs

```diff
@@ -1067,6 +1126,41 @@ mod cmd {
             fun: later,
             completer: None,
         },
+        Command {
+            name: "write-quit",
+            alias: Some("wq"),
+            doc: "Writes changes to disk and closes the current view. Accepts an optional path (:wq some/path.txt)",
+            fun: write_quit,
+            completer: Some(completers::filename),
+        },
+        Command {
+            name: "write-quit!",
+            alias: Some("wq!"),
+            doc: "Writes changes to disk and closes the current view forcefully. Accepts an optional path (:wq! some/path.txt)",
+            fun: force_write_quit,
+            completer: Some(completers::filename),
+        },
+        Command {
+            name: "write-all",
+            alias: Some("wa"),
+            doc: "Writes changes from all views to disk.",
+            fun: write_all,
+            completer: None,
+        },
+        Command {
+            name: "write-all-quit",
+            alias: Some("waq"),
```

Comment:
Let's make these match vim: `wqa`. There's also `qa`/`qall`

Reply:
Done, added `qa` and `qa!`